README.md

```diff
@@ -124,19 +124,15 @@ defaultAddCapabilities: null
 fsGroup:
   type: RunAsAny
 groups:
-priority: 10
```

Comment:
this is already in `develop`... how come it is here?

Reply:
I updated the changes from develop here

Reply:
if this is the state of both develop and your branch then there shouldn't be a diff